packages/@ember/application/lib/application.js

```diff
@@ -205,6 +205,15 @@ const Application = Engine.extend({
   */
   rootElement: 'body',
 
+  /**
+
+    @property rootDocument
+    @type Document
+    @default 'window.document'
+    @public
```

Comment:
@private

Reply:
Wouldn't hurt to make call this field `_document` also to make it clear that it's private